.circleci/config.yml

```diff
@@ -321,7 +321,16 @@ jobs:
         type: string
         default: dev
     steps:
-      - checkout
+      - run:
+          name: Checkout e2e repo
+          command: |
+            git clone https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui.git .
+      - run:
+          name: install
+          command: apt-get install xz-utils
```

Comment:
I'm not 100% sure if these are needed for cypress. CAS3 is the only repo thats using cypress for e2e. CAS1 & CAS2 use playwright. 
This run step was removed from the previous attempt at syncing the configs